vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -67,11 +67,16 @@ def parse_service(service)
     def parse_pod(pod)
       # pod in kubernetes is container group in manageiq
       new_result = parse_base_item(pod)
+      # TODO: https://github.com/GoogleCloudPlatform/kubernetes/issues/3607
```

Comment:
This comment doesn't make it clear what needs "TODO".  It just points to an issue that is already closed.  Can you update the comment to explain that the TODO is to remove this version specific code once old versions are assured to no longer be used?